### PR TITLE
Add CRI-O v1.31 blog post

### PR DESCRIPTION
**Pull request by saschagrunert**
Creating a blog post about CRI-O v1.31.

cc @kubernetes/sig-node-cri-o-test-maintainers 

Preview: https://deploy-preview-47804--kubernetes-io-main-staging.netlify.app/blog/2024/09/18/cri-o-1-31

### CODE REVIEW

**Comment by saschagrunert** [comment]
@kubernetes/sig-node-cri-o-test-maintainers PTAL

/hold
release date pending.

File: content/en/blog/_posts/2024-09-18-cri-o-1.31.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+workloads, for example when it comes to edge use cases as well as running
+WebAssembly (Wasm) workloads.
+
+CRI-O v1.31 also features support for fine-grained `SupplementalGroups`
+([KEP-3619](https://kep.k8s.io/3619)), which allow to control and track how
+supplemental groups are applied to a container process. If you like to learn
+more about the feature itself, then feel free to read through the [corresponding
+Kubernetes v1.31 blog post](/blog/2024/08/22/fine-grained-supplementalgroups-control).
+
+Beside that, the CRI-O maintainers also added support for the Kubernetes image
```
Comment (kannon92) [inline content/en/blog/_posts/2024-09-18-cri-o-1.31.md:34]:
```suggestion
Beside that, the CRI-O community also added support for the Kubernetes image
```

File: content/en/blog/_posts/2024-09-18-cri-o-1.31.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+[namespace](/docs/concepts/overview/working-with-objects/namespaces). This
+means, that policies in the (default) directory
+`/etc/crio/policies/[NAMESPACE].json` will be validated for each pod of the
+corresponding `NAMESPACE`. This will also happen on container creation, which is
+a huge step forward in enforcing sigstore policies for a dedicated Kubernetes
+namespace in comparison to policies which only apply to the whole cluster
+itself.
+
+Beside the support for bigger Kubernetes features, CRI-O v1.31 ships a bunch of
+cool smaller enhancements, for example:
```
Comment (kannon92) [inline content/en/blog/_posts/2024-09-18-cri-o-1.31.md:53]:
```suggestion
other enhancements, for example:
```

Small seems to be short changing these features. Let's not classify them as bigger or smaller.

**Review by kannon92 — COMMENTED** [review]
mostly small nits but content LGTM.

**Comment by sftim** [comment]
Thanks, but this needs to go onto the https://cri-o.io/ website. CRI-O is not part of Kubernetes (see https://www.cncf.io/projects/cri-o/ for supporting evidence). We do however align release numbering between the two projects.

Because https://k8s.io/blog/ definitely, certainly is not the place to put the release announcement, and it's not plausible that this will change ahead of the CRI-O v1.31 release:

/close

If you want to reopen this @saschagrunert, make sure you can convince the CNCF that CRI-O is part of Kubernetes. It'd be a big ask.

**Comment by kwilczynski** [comment]
@sftim, we had a number of blog posts about and around CRI-O accepted in the past, if I am not mistaken.

Have the stance towards CRI-O changed? I am just curious to whether we were doing something wrong in the past and perhaps set some unwelcomed precedent?

**Comment by sftim** [comment]
> Have the stance towards CRI-O changed? I am just curious to whether we were doing something wrong in the past and perhaps set some unwelcomed precedent?

A bit, but these things happen.

Anyway, it could be:
- the articles were about Kubernetes and CRI-O working together
- they were published before CRI-O became a standalone CNCF project
- we made a mistake

**Pull request closed without merging**